cmd/chart-repo/utils.go

```diff
@@ -44,6 +47,8 @@ const (
 	chartCollection       = "charts"
 	chartFilesCollection  = "files"
 	defaultTimeoutSeconds = 10
+	caRegistryDir         = "/etc/registry-ca"
+	rootCARegistry        = "/etc/ssl/certs/ca-certificates.crt"
```

Comment:
I was confused by this. I did not know that this one is the path to the system ca-certificates. I thought that it was the custom rootCA that you want to use.

I'd rename it to systemCA or something like that and mention that you need it since it is required once appending CAs

Reply:
I gave a quick look at Tiller's implementation and it seems that they do not require the systemCAs but instead create an empty pool? https://github.com/helm/helm/blob/7cad59091a9451b2aa4f95aa882ea27e6b195f98/pkg/tlsutil/tls.go#L75